hw6_epam_traning/src/test/java/com/mapLine/MapCreatorTest.java

```diff
@@ -0,0 +1,45 @@
+package com.mapLine;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.TreeMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class MapCreatorTest {
+
+    private  Map<Character, Integer> wordsMap;
+
+    @BeforeEach
+    public void init() {
+        wordsMap = new TreeMap(Collections.reverseOrder());
+    }
+
+    @Test
+    void sortMap_whenPutLine() {
+        wordsMap.put('A', 1);
```

Comment:
All stages should be separate with an empty line like this: 

        wordsMap.put('2', 4);
        wordsMap.put('3', 1);
        String value = "23 2 2 2";
        
        assertThat(wordsMap).isEqualTo(MapCreator.mapCreator(value));

Reply:
Or using 
```
               //given

               //when

              //then
```